_posts/2020-09-23-#19761.md

```diff
@@ -10,14 +10,44 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Bitcoin Core requires several
+  [dependencies](https://github.com/bitcoin/bitcoin/blob/master/doc/dependencies.md).
+  These include large dependencies such as [boost](https://www.boost.org/),
+  [libevent](https://libevent.org/) and [qt](https://libevent.org/), as well as
+  several smaller dependencies.
+
+- These dependencies can be built locally as part of a [depends
+  build](https://github.com/bitcoin/bitcoin/blob/master/depends/README.md).
+
+- The versions of the dependency libraries used is pinned by a specific hash
+  (eg
+  [here](https://github.com/bitcoin/bitcoin/blob/652c45fdbbd55bde95c8c6cf08a5feb6055ac112/depends/packages/bdb.mk#L5)
+  for bdb).
+
+- Sometimes small changes need to be made to the library before building. This
+  has previously been done using [sed](https://en.wikipedia.org/wiki/Sed). sed
```

Comment:
- suggest s/sed/`sed`/ for the second "sed" that starts the third sentence (as well as for `sed` in the following bullet point); `sed` is already in `code` format in the questions

- and/or start a new bullet point for the third sentence, for readability